src/atomizer.js

```diff
@@ -122,8 +122,8 @@ Atomizer.prototype.findClassNames = function (src/*:string*/)/*:string[]*/ {
  *
  * getConfig(['Op(1)', 'D(n):h']);
  */
-Atomizer.prototype.getConfig = function (classNames/*:string[]*/, config/*:AtomizerConfig*/)/*:AtomizerConfig*/ {
-    config = config || { classNames: [] };
+Atomizer.prototype.getConfig = function (classNames/*:string[]*/, existingConfig/*:AtomizerConfig*/)/*:AtomizerConfig*/ {
+    const config = existingConfig && _.cloneDeep(existingConfig) || { classNames: [] };
```

Comment:
I was in two minds about this approach, I thought we could avoid a deep clone but actually this is nice because no one consuming this can't mistakenly mutate a reference in the original config via the new one either.

Seems like the right way to go to me 👍